src/app/home/home.hbs

```diff
@@ -0,0 +1,33 @@
+  <section class="home-container">
+    <div class="slideshow-container">
+        {{#each banners.banners}}
+        <div class="bannerSlides fade">
+                <img src="{{bannerImageUrl}}" style="width:100%" alt="{{bannerImageAlt}}">
+        </div>
+        {{/each}}
+      <a class="prev">&#10094;</a>
```

Comment:
Anchor without Href is accessibility's P1 defect

Reply:
Updated.